dev-util/nvidia-cuda-sdk/nvidia-cuda-sdk-9.1.85.ebuild

```diff
@@ -0,0 +1,144 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cuda eutils flag-o-matic portability toolchain-funcs unpacker versionator
```

Comment:
do you need eutils here?

Reply:
I assume it can be dropped and is a leftover from previous bumps where user defined patches where applied.

Reply:
Actually, `eapply_user` is still in, as user might wan't to apply their own patches. However, no functions of the `eutils.eclass` are used any more, therefore I dropped it within an additional commit.